app/Http/Controllers/Rest/CampaignsController.php

```diff
@@ -101,30 +93,27 @@ public function count(Request $request)
             'averageCPC' => 0,
             'totalCost' => 0,
         ];
-        $response = self::json($siteCount, 200);
 
-        return $response;
+        return self::json($siteCount, 200);
     }
 
     public function edit(Request $request, $campaign_id)
     {
-//        $this->validateRequestObject(
-//            $request,
-//            'campaign',
-//            array_intersect_key(
-//                Campaign::$rules,
-//                $request->input('campaign')
-//            )
-//        );
+        $this->validateRequestObject(
+            $request,
+            'campaign',
+            array_intersect_key(
+                Campaign::$rules,
+                $request->input('campaign')
+            )
+        );
 
         // TODO check privileges
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)